src/Storefront/Resources/views/storefront/component/address/address-editor-modal.html.twig

```diff
@@ -1,3 +1,6 @@
+{# @deprecated tag:v6.7.0 - File will be removed as it is no longer loaded by default. #}
+{# Extend `src/Storefront/Resources/views/storefront/component/address/address-manager-modal.html.twig` instead #}
+
 {% set defaultBillingAddressId = context.customer.defaultBillingAddressId %}
 {% set defaultShippingAddressId = context.customer.defaultShippingAddressId %}
 
```

Comment:
The rest of the file is pretty hard to read. As far as I can tell, only the formatting has changed, so I think it would be best if you changed it back and don't touch it.